lib/ash_json_api/controllers/router.ex

```diff
@@ -96,7 +96,7 @@ defmodule AshJsonApi.Controllers.Router do
   end
 
   defp open_api_request?(conn, open_api) do
-    AshJSonApi.OpenApiSpexChecker.has_open_api?() && conn.method == "GET" &&
+    AshJsonApi.OpenApiSpexChecker.has_open_api?() && conn.method == "GET" &&
```

Comment:
The uppercase "S" in AshJSonApi was inconsistent with the rest of the project. Renamed for consistency.